### DISCUSSION TITLE
Kilitler ve anahtarlar

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#55] [Zeka Sorusu]
7 korsan bir hazine buluyor ve bütün paraları büyükçe bir sandığa koyuyor. Bu sandığı kilitlemek istiyorlar ama bunu öyle yapmak istiyorlar ki sandığı ancak en az 4 korsan beraber açabilsin. Herhangi 3 korsan açamayacak ama herhangi 4, 5, 6 korsan ya da hepsi birden açabilecek. Her kilidin birden fazla anahtarı olabilir ama bir anahtar sadece bir kilidi açabilir. Bu durumda sandığı kilitlemek için en az kaç kilit gerekir? Korsanlar kaç anahtar almalı?

**Comment by Deniz34000** [comment]
en az 4 korsan açabildiğine ve her kilidin bir anahtarı olabildiğine göre 4 anahtar yeterli

**Comment by Sevvalgocgun1** [comment]
Bu sorunun cevabı şu şekildedir:

En az 7 kilit gerekir.

Her korsan 3 anahtar almalıdır.
